CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Lotus changelog
 
+# 1.14.3 / 2022-03-03
```

Comment:
This should be 1.14.4.

Wanna add the notes for 1.14.3 while you're at it?

Reply:
nice catch! and yes